Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM golang:1.18
+FROM golang:1.18 as builder
 
-RUN mkdir /app
 WORKDIR /app
-
-COPY go.mod ./
-COPY go.sum ./
-RUN go mod download
```

Comment:
I don't think removing those steps is a good idea. Basically, it removes quickly caching if dependencies didn't change.

Reply:
you're right